test/parallel/test-https-agent-disable-session-reuse.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
 const assert = require('assert');
 const https = require('https');
-const fs = require('fs');
 
 const TOTAL_REQS = 2;
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fixtures.readKey('/keys/agent1-key.pem'),
+  cert: fixtures.readSync('/keys/agent1-cert.pem')
```

Comment:
The keys directory should not be used anymore. You only need the filename.

Reply:
These are also using two different methods... it should be:

```js
key: fixtures.readKey('agent1-key.pem'),
cert: fixtures.readKey('agent1-cert.pem')
```

Reply:
oops! didn't see that @joyeecheung had already pointed that out below :-)